src/js/popup.js

```diff
@@ -58,6 +61,29 @@
       alertListContainer.addClass('is-hidden');
   }
 
+  function getNowValue() {
+    nowValueService.currentValue()
+      .then(updateNowAndVariation);
+  }
+
+  function updateNowAndVariation(response){
+    var value = response.dolar.cotacao.substring(0,4);
+    var variation = response.dolar.variacao.substring(0,5);
+
+    if(isPositiveVariation(variation)) {
+      nowValueVariation.addClass('is-positive');
+    } else {
+      nowValueVariation.addClass('is-negative');
+    }
+
+    nowValueField.text(value);
+    nowValueVariation.text(variation);
+  }
+
+  function isPositiveVariation(variation) {
+    return variation.charAt(0) == '+';
```

Comment:
Didn't know that javascript would parse that as a positive and negative depending on '+' or  '-' signals. That's neat.